ops/terraform/env/mgmt/alarms.tf

```
@@ -106,3 +111,39 @@ resource "aws_cloudwatch_metric_alarm" "ec2_failing_instances" {
   alarm_actions = [aws_sns_topic.victor_ops_alert.arn]
   ok_actions    = [aws_sns_topic.victor_ops_ok.arn]
 }
+
+## BFD-3520
+resource "aws_cloudwatch_log_metric_filter" "ec2_init_fail_count" {
+  name           = local.init_fail_filter_name
+  pattern        = local.init_fail_pattern
+  log_group_name = local.log_groups.cloudinit_out
+
+  metric_transformation {
+    name          = local.init_fail_metric_name
+    namespace     = local.this_metric_namespace
+    value         = "1"
+    default_value = "0"
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "ec2_init_fail" {
+  alarm_name          = local.init_fail_alarm_name
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = "1"
+
+  metric_name = local.init_fail_filter_name
+  namespace   = local.this_metric_namespace
+  period      = 60
+
+  statistic = "Maximum"
+  threshold = "0"
+
+  alarm_actions = [aws_sns_topic.victor_ops_alert.arn]
+
+  actions_enabled = true
+  alarm_description = join("", [
+    "At least 1 (see Alarm value for exact number) EC2 instance is failing startup steps.\n",
+    "See ${local.ec2_instance_script_failing_start_runbook_url} for instructions on resolving this alert."
```

nit: In the interim between BFD-3553 being complete and BFD-3520 being merged it's probably best if this Alarm doesn't link to a Runbook for a _different_ class of issue. If someone is (unfortunately) paged due to this Alarm firing, we definitely don't want them to go down the rabbit hole of following the instructions on that Runbook only to be confused when they are unable to determine any failing EC2 Instances.

I'm leaving this as a "nit" because I think BFD-3553 will be done fairly quickly, so leaving this around for a bit longer probably is fine. 

```suggestion
    # TODO: Introduce Runbook URL for determining failing instances in BFD-3553
    "At least 1 (see Alarm value for exact number) EC2 instance is failing startup steps."
```

---

applied as requested